swaap/swaap/View Controllers/ContactsViewController.swift

```diff
@@ -100,7 +138,11 @@ class ContactsViewController: UIViewController, ProfileAccessor, ContactsAccesso
 
 extension ContactsViewController: UITableViewDelegate, UITableViewDataSource {
 	func numberOfSections(in tableView: UITableView) -> Int {
-		fetchedResultsController.sections?.count ?? 0
+		fetchedResultsController.sections?.count ?? 1
+	}
+
+	func sectionIndexTitles(for tableView: UITableView) -> [String]? {
+		fetchedResultsController.sectionIndexTitles
```

Comment:
not sure if this is necessary

Reply:
Again, this is from trying to implement indexing. I'll get rid of these

Reply:
I think section titles are a good idea, but that’s the other implemented function I think. This one can either be removed or finish working on the indices imo.